_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -6586,11 +6586,6 @@
 /obj/effect/decal/cleanable/dirt,
 /turf/open/floor/plating,
 /area/maintenance/tram/left)
-"aMM" = (
-/obj/structure/lattice/catwalk,
-/obj/effect/decal/cleanable/dirt,
-/turf/open/floor/plating,
-/area/maintenance/tram/left)
```

Comment:
This is dumb. If they look wrong that's a failing of the system. rather then something to be hid

Reply:
These changes were removed.